packages/strapi-design-system/src/Select/Select.js

```diff
@@ -0,0 +1,196 @@
+import React, { Children, cloneElement, useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import DropdownIcon from '@strapi/icons/FilterDropdownIcon';
+import CloseAlertIcon from '@strapi/icons/CloseAlertIcon';
+import { SelectButton } from './SelectButton';
+import { Field, FieldHint, FieldLabel, FieldError } from '../Field';
+import { Popover } from '../Popover';
+import { Stack } from '../Stack';
+import { Text } from '../Text';
+import { Row } from '../Row';
+import { genId } from '../helpers/genId';
+import { SelectList } from './SelectList';
+import { SelectButtonWrapper, IconBox, CaretBox } from './components';
+
+export const Select = ({
```

Comment:
Do you think that we can consider passing an `options` prop as well like a HTML select?

```
const options = [ { label: String, value: Object|String|Number } ];
```

Reply:
Nope, it will be up to the host application. I'm a bit against binding a specific data shape to a UI component for this purpose. Since you can do exactly the same thing on render when mapping over the "<Option>" components